src/inmanta/data/__init__.py

```diff
@@ -2377,6 +2377,12 @@ def convert_agent_trigger_method(value: object) -> str:
     return value
 
 
+def validate_cron_or_int(value: Union[int, str]) -> str:
+    if isinstance(value, int):
```

Comment:
The type defined in the definition of the setting is `str`. This will also be mentioned like this on our documentation pages. But if the user sends an integer value as a string to the API, this condition will not match and the validation will fail.